code/game/objects/items/stacks/stack.dm

```diff
@@ -236,8 +236,13 @@
 	return
 
 /obj/item/stack/attackby(obj/item/W, mob/user, params)
-	if(istype(W, src.type))
+	if(istype(W, src.type) || (istype(W, /obj/item/stack/cable_coil) && istype(src, /obj/item/stack/cable_coil)))
```

Comment:
It seems as though if W type was `/obj/item/stack/cable_coil` or `/obj/item/stack/cable_coil/cyborg` and src type was `/obj/item/stack/cable_coil/random` it wouldn't match